### PR TITLE
⬆️ Upgrades Z-Wave JS UI to v8.8.6

### PR DESCRIPTION
**Pull request by roblandry**
# Proposed Changes
Upgrades Z-Wave JS UI to v8.8.5
  * https://github.com/zwave-js/zwave-js-ui/releases/tag/v8.8.4
  * https://github.com/zwave-js/zwave-js-ui/releases/tag/v8.8.5
  * https://github.com/zwave-js/zwave-js-ui/releases/tag/v8.8.6

### CODE REVIEW

File: zwave-js-ui/Dockerfile [hunk 1]
```diff
@@ -12,18 +12,18 @@ RUN \
...
         npm=9.1.2-r0 \
-        python3-dev=3.10.9-r1 \
+        python3-dev=3.10.10-r0 \
     \
     && apk add --no-cache \
         eudev=3.2.11-r4 \
         libusb=1.0.26-r0 \
         nginx=1.22.1-r0 \
-        nodejs=18.12.1-r0 \
+        nodejs=18.14.1-r0 \
```
Comment (frenck) [inline zwave-js-ui/Dockerfile:21]: These changes are not related to this PR/bump and should thus not be in this PR.

**Review by frenck — CHANGES_REQUESTED** [review]

File: zwave-js-ui/Dockerfile (hunk 1 above)
Comment (roblandry) [reply zwave-js-ui/Dockerfile:21]: Unfortunately the build will not complete without these updates. The previous packages do not exist in alpine 3.17, and the build fails.
https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.17&repo=&arch=&maintainer=
https://pkgs.alpinelinux.org/packages?name=python3-dev&branch=v3.17&repo=&arch=&maintainer=

Comment (crawford) [reply zwave-js-ui/Dockerfile:21]: Can you fix the dependencies in one PR (getting the pipeline working again) and then follow up with the bump in another?

Comment (roblandry) [reply zwave-js-ui/Dockerfile:21]: Yeah, when I get off work I'll submit a separate PR for the dependencies and update this one to only 8.8.5.

Comment (roblandry) [reply zwave-js-ui/Dockerfile]: Done. 
#470 
[7f9ae6f](https://github.com/hassio-addons/addon-zwave-js-ui/pull/467/commits/7f9ae6ff951ee25e2cb9cf1ee3454a043e06168a)

File: zwave-js-ui/Dockerfile [hunk 2]
```diff
@@ -23,7 +23,7 @@ RUN \
     && npm install --global yarn@2.4.3 \
     \
     && curl -J -L -o /tmp/zwave-js-ui.tar.gz \
-        "https://github.com/zwave-js/zwave-js-ui/archive/v8.8.3.tar.gz" \
+        "https://github.com/zwave-js/zwave-js-ui/archive/v8.8.5.tar.gz" \
```
Comment (frenck) [inline zwave-js-ui/Dockerfile:26]:
```suggestion
        "https://github.com/zwave-js/zwave-js-ui/archive/v8.8.6.tar.gz" \
```

**Review by frenck — APPROVED** [review]
Thanks, @roblandry 👍 

../Frenck

**Pull request merged**